packages/expo-calendar/src/Calendar.ts

```diff
@@ -1355,6 +1355,37 @@ export async function requestCalendarPermissionsAsync(): Promise<PermissionRespo
   return await ExpoCalendar.requestCalendarPermissionsAsync();
 }
 
+// @needsAudit
+/**
+ * Asks the user to grant write-only permissions for accessing and modifying user's calendars.
+ * This is a more limited scope than full calendar access, allowing only event creation and modification.
+ *
+ * @return A promise that resolves to an object of type [`PermissionResponse`](#permissionresponse).
+ * @platform ios
+ */
+export async function requestCalendarWritePermissionsAsync(): Promise<PermissionResponse> {
+  if (!ExpoCalendar.requestCalendarWritePermissionsAsync) {
```

Comment:
we don't really need this check anymore, but let's keep it for consistency for now i guess